src/configuration.ts

```diff
@@ -16,7 +16,10 @@
 
 import * as fs from "fs";
 import * as path from "path";
-import * as findup from "findup-sync";
+
+// hack to correctly import JS module that uses module.exports = function(){...}
+import * as findupSync from "findup-sync";
+const findup = (<any>findupSync).default;
```

Comment:
@weswigham Do you know if there's a better way to do this when targeting ES6 and then using Babel?